src/main/java/upbrella/be/user/controller/UserController.java

```diff
@@ -174,4 +175,21 @@ public ResponseEntity<CustomResponse> readUserHistories(HttpSession session) {
                         rentService.findAllHistoriesByUser(loginedUserId)
                 ));
     }
+
+    @PatchMapping("/bankAccount")
+    public ResponseEntity<CustomResponse> updateUserBankAccount(HttpSession session, UpdateBankAccountRequest updateBankAccountRequest) {
```

Comment:
UpdateBankAccountRequest에 @NotBlank 등의 유효성 검증을 위한 어노테이션을 사용하셨는데 이를 검증하기 위해서 컨트롤러의 메서드 파라미터에 @Valid 어노테이션은 사용하지 않아도 유효성 검증이 되는건가요?
```suggestion
    @PatchMapping("/bankAccount")
    public ResponseEntity<CustomResponse> updateUserBankAccount(HttpSession session, @Valid UpdateBankAccountRequest updateBankAccountRequest) {
```